scenedetect/video_splitter.py

```diff
@@ -227,15 +227,14 @@ def split_video_ffmpeg(input_video_paths, scene_list, output_file_template, vide
             call_list += [
                 '-y',
                 '-ss',
-                str(start_time.get_seconds()),
+                str(start_time.get_timecode()),
```

Comment:
Just curious, why did you use get_timecode here as opposed to get_seconds?

Reply:
49c3258e0db74c38e97aa467f83ad830da9d28d0 is the reason I ask.

Reply:
Interesting.. There wasn't any particular reason, I simply find it neater, I didn't notice any difference in my testing. I'll swap it back, now that I'm using duration as well, makes sense for both to be in seconds